pkg/gce-pd-csi-driver/node.go

```diff
@@ -461,6 +461,20 @@ func (ns *GCENodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpa
 		return nil, status.Error(codes.Internal, fmt.Sprintf("error when getting device path for %s: %v", volumeID, err))
 	}
 
+	volumeCapability := req.GetVolumeCapability()
+	if volumeCapability != nil {
+		// VolumeCapability is optional, if specified, validate it
+		if err := validateVolumeCapability(volumeCapability); err != nil {
+			return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("VolumeCapability is invalid: %v", err))
+		}
+
+		if blk := volumeCapability.GetBlock(); blk != nil {
```

Comment:
Can you add a unit test for this?

Reply:
The `TestNodeExpandVolume` unit tests are currently broken and require more extensive work to fix: see https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/573